ui/src/config/section/storage.js

```diff
@@ -165,9 +165,10 @@ export default {
           label: 'label.action.take.snapshot',
           dataView: true,
           show: (record, store) => {
-            return record.state === 'Ready' && (record.hypervisor !== 'KVM' ||
-                record.hypervisor === 'KVM' && record.vmstate === 'Running' && store.features.kvmsnapshotenabled ||
-                record.hypervisor === 'KVM' && record.vmstate !== 'Running')
+            return record.state === 'Ready' &&
+              (record.hypervisor !== 'KVM' ||
+               ['Stopped', 'Destroyed'].includes(record.vmstate) ||
```

Comment:
same here, why destroyed state ?

Reply:
Same as above

Reply:
Before this change, the button would be shown for all states part from 'Running'.
Now it will only be shown for 'Stopped' or 'Destroyed' states